<!--
	Launch file for testing purposes
	
	After launching this file, it is necessary to publish the
	acquisition command. This can be done as below:
		rostopic pub -1 /RH/cmd/acquire rh_cameras/CamerasSync '{timeStamp: now, data: full}'
		
	SoCS, UoG. Gerardo Aragon-Camarasa and Susanne B Oehler. August 2012 
-->

<launch>

	<!-- Include Robot Head parameter server configuration. -->
    <include file="$(find clopema_config)/launch/clopema_$(optenv CLOPEMA_PARTNER error).launch" />
	<include file="$(find rh_launch)/launch/RH_parameters_$(optenv CLOPEMA_PARTNER error).launch" />
	
	<!-- Image list file -->
    <param name="/RH/cameras/image_list" value="$(find rh_calibration)/src/input_images.xml" />
	
	<node pkg="rh_cameras" type="RHcam_simulation" name="RHcam_simulation" output="screen">
	</node>
	
	<node pkg="rh_vergence" type="RHvergence_simulation" name="RHvergence_simulation" output="screen">
	</node>
	
</launch>

